x/programs/cmd/simulator/cmd/root.go

```diff
@@ -89,7 +91,7 @@ func NewRootCmd() *cobra.Command {
 func (s *simulator) Init() error {
 	homeDir, err := os.UserHomeDir()
 	if err != nil {
-		return nil
+		return err
```

Comment:
Interesting that we didn't have a linter catch this 🤔

Reply:
This was all @darioush when he was helping me debug

Reply:
Created an issue
https://github.com/ava-labs/hypersdk/issues/752